packages/build-config/src/index.ts

```diff
@@ -102,8 +102,9 @@ function recastMacrosConfig(macros: object): MacrosWithGlobalConfig {
   return macros as MacrosWithGlobalConfig;
 }
 
+export function setConfig(macros: object, config: WarpDriveConfig): void;
 export function setConfig(context: object, appRoot: string, config: WarpDriveConfig): void {
```

Comment:
In order to use function overloading you need both signatures AND the implementation signature.  

```ts
setConfig(macros: object, config: WarpDriveConfig): void;
setConfig(context: object, appRoot: string, config: WarpDriveConfig): void;
export function setConfig(context: object, appRootOrConfig: string | WarpDriveConfig, config?: WarpDriveConfig): void {
```

Reply:
yes, but I was lazy and only used the web ui haha

Reply:
coulda still be done :P